op-chain-ops/addresses/roles.go

```diff
@@ -28,3 +36,21 @@ type OpChainFaultProofsRoles struct {
 	Proposer   common.Address
 	Challenger common.Address
 }
+
+var ErrSuperchainRoleZeroAddress = errors.New("SuperchainRole is set to zero address")
+
+func (s *SuperchainRoles) CheckNoZeroAddresses() error {
+	val := reflect.ValueOf(*s)
+	typ := reflect.TypeOf(*s)
+
+	// Iterate through all the fields
+	for i := 0; i < val.NumField(); i++ {
```

Comment:
Tiny thing but for i := range(val.NumField()) works too

Reply:
Functionally that is equivalent. I just have a slight preference to the current implementation because its more clear what its doing (at least to me)